### PR TITLE
Merge spotter

### PR DESCRIPTION
**Pull request by desaperados**
Move the Spotter + commit history into dss:
```
github/makerdao/dss-deploy/src/poke.sol -> src.spot.sol
```

### CODE REVIEW

**Comment by rainbreak** [comment]
@gbalabasquer I like this. How about you?

**Comment by gbalabasquer** [comment]
@rainbreak same here, like it too.

File: src/test/spot.t.sol [hunk 1]
```diff
@@ -0,0 +1,9 @@
+pragma solidity >=0.5.0;
+
+import "ds-test/test.sol";
+
+import {Spotter} from "../spot.sol";
+
+contract SpotterTest is DSTest {
```
Comment (rainbreak) [inline src/test/spot.t.sol]: Is there any point having this stubbed?

Comment (desaperados) [reply src/test/spot.t.sol]: I guess it's pretty redundant. Reasoning was that it highlights lack of tests and suggests that we write some, but not a considered choice to include it - happy to remove.

**Pull request merged**